content/en/docs/Examples/services.md

```diff
@@ -114,6 +114,24 @@ service:
       icon: https://raw.githubusercontent.com/dani-garcia/vaultwarden/main/src/static/images/vaultwarden-icon.png
 ```
 
+## Fallenbagel/jellyseerr
+Source: https://github.com/Fallenbagel/jellyseerr
+```yaml
+Fallenbagel/jellyseerr:
+  latest_version:
+    type: github
+    url: Fallenbagel/jellyseerr
+    url_commands:
+    - type: regex
+      regex: v([0-9.]+)$
```

Comment:
```suggestion
    url_commands:
      - type: regex
        regex: v([0-9.]+)$
```

Thanks for doing these, happy to add them :). This list could do with indenting to match the others

Reply:
Ah my bad, missed those again. I have a habit of indenting them this way.